diff --git a/third_party/py/rules_pywrap/pywrap.impl.bzl b/third_party/py/rules_pywrap/pywrap.impl.bzl
index 7c663a2..1f80949 100644
--- a/third_party/py/rules_pywrap/pywrap.impl.bzl
+++ b/third_party/py/rules_pywrap/pywrap.impl.bzl
@@ -2,6 +2,7 @@ load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain", "use_c
 load("@rules_cc//cc:cc_binary.bzl", "cc_binary")
 load("@rules_cc//cc:cc_import.bzl", "cc_import")
 load("@rules_cc//cc:cc_library.bzl", "cc_library")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
 load("@rules_python//python:py_info.bzl", "PyInfo")
 load("@rules_python//python:py_library.bzl", "py_library")
 
